src/platform/i386/kernel.h

```diff
@@ -25,16 +25,27 @@ void console_init(void);
 void serial_init(void);
 #endif
 
-#ifdef ENABLE_TIMER
-void timer_init(u32_t frequency);
+enum {
```

Comment:
`typedef` this to `timer_flag_t` or something else.  We want this to be self-documenting in the APIs.